ui/desktop/electron-app/src/handlers.js

```diff
@@ -20,51 +20,25 @@ handle('getOrigin', () => origin);
  */
 handle('setOrigin', (requestOrigin) => origin = requestOrigin);
 
+/**
+ * Detect boundary cli path
+ */
+handle('cliPath', async () => {
+  const cliPath = await lookpath('boundary');
+  if(!cliPath) throw new Error({ message: 'Cannot find boundary cli.'})
```

Comment:
Two things here:

- The targets route throws its own, duplicate error for this.  Ideally it would only need to be thrown in one place.
- Should this method return a boolean?  It makes a bit more sense to me that it would just be a boolean `true` / `false`.  Either the CLI is found or it's not.

Reply:
Removed cli path errors from handler. It makes sense for path checks to happen on the side of things that invoked it.
In addition, separated out 'exists' and 'path' functions for convenience.